addons/stock_account/models/stock_quant.py

```diff
@@ -61,6 +61,12 @@ def _read_group(self, domain, groupby=(), aggregates=(), having=(), offset=0, li
 
         return result
 
+    @api.model
+    def read_group(self, domain, fields, *args, **kwargs):
```

Comment:
Please use `_read_group_select`  instead for this kind of override like in stock 

https://github.com/odoo/odoo/blob/ceaffb371a7d31d92e00a84841421c8f89b97d11/addons/stock/models/stock_quant.py#L299-L308

Reply:
Hello @Whenrow , we overrided the ``read_group`` because when trying with ``_read_group_select`` it was never called for the computed field ``value``